Example/SnapshotTests/SnapshotTestingTests.swift

```diff
@@ -138,6 +138,32 @@ final class SnapshotTestingTests: XCTestCase {
         assertSnapshot(matching: viewController, as: .imageWithSmartInvert, named: nameForDevice())
     }
 
+    @available(iOS 16.0, *)
```

Comment:
If I remember correctly this doesn't do quite enough to make the tests pass and I'll need to add an `XCTSkip`.